frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -450,6 +450,31 @@ class RegistrationForm extends Component {
     );
   };
 
+  // analyses field by field and focus on the highest error field
+  focusOnHighestErrorField = () => {
+    if (!this.state.isValidFirstName) {
```

Comment:
part of me feels like this should be a switch statement, but I'm ok with it like this.

Reply:
Ignore me, I'm crazy